webapp/src/actions/index.js

```diff
@@ -5,19 +5,11 @@ import {PostTypes} from 'mattermost-redux/action_types';
 
 import Client from '../client';
 
-export function startMeeting(channelId, rootId, force = false, topic = '') {
+export function startMeeting(channelId, rootId = '', force = false, topic = '') {
     return async (dispatch, getState) => {
         try {
-            let meetingURL = '';
-            if (force) {
-                meetingURL = await Client.forceStartMeeting(
-                    channelId, true, '', 0, topic,
-                );
-            } else {
-                meetingURL = await Client.startMeeting(
-                    channelId, true, '', 0, false, topic,
-                );
-            }
+            const startFunction = force ? Client.forceStartMeeting : Client.startMeeting;
+            const meetingURL = await startFunction(channelId, rootId, true, topic);
```

Comment:
Looks good for this PR. Unrelated comment below

---

Do we really need a `Client.forceStartMeeting`? Since that function really just calls `Client.startMeeting` with `force=true`

Also, this function signature is pretty hard to read, along with the function call below. Thoughts on changing this to accept an object instead? Would be best if it were typescript as well so we have type safety on the object

```ts
type StartMeetingArgs = {
    channelId: string;
    rootId?: string;
    force?: boolean;
    topic?: string;
}

export function startMeeting(args: StartMeetingArgs) {
    return async (dispatch, getState) => {
        try {
            const startFunction = force ? Client.forceStartMeeting : Client.startMeeting;
            const meetingURL = await startFunction(channelId, rootId, true, topic);
```

We can do this as a follow up thing since that would expand the scope of this fix quite a bit

Reply:
@mickmister I agree with your suggestions. Should I create an issue for the same?

Reply:
@raghavaggarwal2308 Yes sounds good to me :+1:

Reply:
@mickmister Created the issue. Please let me know if we need to add anything else.
https://github.com/mattermost/mattermost-plugin-zoom/issues/319